### DISCUSSION TITLE
矩形数量太大，导致拖拽慢，有没有好的办法解决拖动卡顿的问题

### DISCUSSION THREAD

**Discussion by zhangluluaichimi** [didi/LogicFlow#1387] [Q&A]
矩形图代表的是展位，该矩形是自定义并修改过后的矩形，但是数据一点多了起来，卡顿太严重
![1698221458872](https://github.com/didi/LogicFlow/assets/51119207/bb65fc0b-c05b-4a14-91cb-aee73fdd89cc)

**Comment by wumail** [comment]
目前没什么好办法，如果是同时操作这么多元素，卡顿本身就是符合预期的，如果操作单个元素也卡，考虑是logicflow底层的问题，需要后面专门进行排查和优化，暂时没有相关的计划

**Comment by wumail** [comment]
信息太少了，没办法告诉你可能可以怎么优化，举个例子，你使用的是普通的矩形还是可缩放的矩形？如果是可缩放的矩形可不可以更改为普通的矩形，将通过交互改变矩形大小的方式改为通过在属性面板中进行修改。
我之前测试过，对于单纯的矩形，几千个的情况下卡顿情况也都在可接受范围内。
